lib/gli/commands/help_modules/command_help_format.rb

```diff
@@ -13,38 +13,38 @@ def initialize(command,app,sorter,synopsis_formatter_class,wrapper_class=TextWra
         end
 
         def format
-          command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
-          wrapper              = @wrapper_class.new(Terminal.instance.size[0],4)
+          @command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
```

Comment:
It was probably due to the ERB template. I don't know if you read Metaprogramming Ruby, but at this book, the author suggests using a "clean room" technique in such situations. You create a class, with all attributes needed inside the template and pass its binding to ERB. This should be cleaner, but also would need a lot more code change. Baby steps.